lib/features/login/presentation/bloc/login_bloc.dart

```diff
@@ -10,11 +10,7 @@ class LoginBloc extends Bloc<LoginEvent, LoginState> {
   LoginBloc(LoginState initialState) : super(initialState);
 
   @override
-  LoginState get initialState => LoginInitial();
-  @override
-  Stream<LoginState> mapEventToState(
-    LoginEvent event,
-  ) async* {
+  Stream<LoginState> mapEventToState(LoginEvent event) async* {
```

Comment:
```suggestion
  Stream<LoginState> mapEventToState(
  LoginEvent event,
  ) async* {
```

Reply:
`Trailing comma.` -> as we talked over slack, this is a method signature in Dart, not Flutter code (widgets). So the trailling commas actually makes it harder to read than without it (see your suggested change and observe how having mutiple lines makes it hard to read the method name. :/

`are we still using this?` -> no we are not... but I swear that I will once we start talking to the API for proper authentication 😇